ofl/signika/DESCRIPTION.en_us.html

```diff
@@ -1,12 +1,18 @@
 <p>
-Signika is a sans-serif with a gentle character, developed for wayfinding, signage, and other media where clarity of information is required. 
-It has a low contrast and tall x-height to improve readability of texts in small sizes as well as in large distances from the reader. 
-Being a typical signage typeface it is inspired by typefaces such as Ronnia, Meta, and Tahoma.
+    Signika is a sans-serif with a gentle character, developed
+    for wayfinding, signage, and other media where clarity of information is required.
+    It has a low contrast and tall x-height to improve readability of texts in small sizes as well as in large distances
+    from the reader.
+    Being a typical signage typeface it is inspired by typefaces such as Ronnia, Meta, and Tahoma.
 </p>
 <p>
-The typeface comes with a wide character set supporting Western European languages, Polish, and Czech. 
-The figures are designed as tabular.
+    The typeface comes with a wide character set supporting Western European languages, Polish, and Czech.
+    The figures are designed as tabular.
 </p>
 <p>
-<a href="https://fonts.google.com/specimen/Signika+Negative">Signika Negative</a> is an alternative version, optimized to solve the effect of juxtaposed positive and negative text setting, where the text in negative tends to look thicker.
+    Signika Negative is an alternative version, optimized to solve the effect of juxtaposed positive and negative text
+    setting, where the text in negative tends to look thicker.
 </p>
+<p>
+    The <a href="https://github.com/Ancymonic/Signika">source code</a> of Signika is available from Github.
+</p>
```

Comment:
Please add EOL.

Reply:
PR branch amended, cheers for the pointer.